src/mpyl/steps/deploy/k8s/resources/dagster.py

```diff
@@ -82,9 +82,12 @@ def to_user_code_values(
                         "repository": f"{docker_registry.host_name}/{project.name}",
                     },
                     "labels": {
-                        k: v
-                        for k, v in builder.to_labels().items()
-                        if not k.startswith("app.")
+                        **{
```

Comment:
do you need the `**`?

Reply:
I don't know, but anything else and it breaks my linter locally and copilot suggests this 😞